Items Editing/az_Smart split items by mouse cursor.lua

```diff
@@ -1,7 +1,10 @@
 -- @description Smart split items using mouse cursor context (also edit cursor, razor area and time selection)
 -- @author AZ
--- @version 3.15
--- @changelog - fixed one silly bug for new users
+-- @version 3.2
```

Comment:
Version numbers are not decimal numbers. 3.2 is a lot smaller (13 minor versions smaller) than 3.15 and so users would not receive the update.

Reply:
Ah, so will 3.20 work fine?

Reply:
Yes. Should I change it to 3.20 and merge?

Reply:
Yes, please)